src/frontend/screens/Settings/components/__tests__/Gamescope.test.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { act, ReactNode, useState } from 'react'
+import { render, screen } from '@testing-library/react'
+import userEvents from '@testing-library/user-event'
+import '@testing-library/jest-dom/extend-expect'
+import Gamescope from '../Gamescope'
+import ContextProvider from 'frontend/state/ContextProvider'
+import { ContextType, SettingsContextType } from 'frontend/types'
+import SettingsContext from '../../SettingsContext'
+
+// TODO: Move this somewhere else once we make it more generic
+const SettingsWrapper = (props: React.PropsWithChildren) => {
```

Comment:
in the future we can probably move this somewhere else and use a factory/generator to create the wrapper with the desired initial state